config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   devise_for :users
   root 'games#index'
-  resource :games
+  resource :games do
```

Comment:
We discussed this over Slack, we'll need to change this to utilize the plural resource format so all you'll need to do here is add an 's'

`resources :games do`